app/views/ThreadMessagesView/SearchHeader.js

```diff
@@ -21,7 +21,8 @@ const styles = StyleSheet.create({
 });
 
 // TODO: it might be useful to refactor this component for reusage
```

Comment:
I left this comment for a moment like this.
We need to refactor this component and make it reusable instead of adding coupling.